### PR TITLE
feat: migrate home page

### PR DESCRIPTION
**Pull request by adibfirman**
Migrating a home page, and create a new component and its called `LinkButton`

### CODE REVIEW

**Comment by adibfirman** [comment]
@resir014 yang lihat selengkapnya ku belum masukin path `/learning` sih it's okay, isn't it?

File: pages/index.tsx [hunk 1]
```diff
@@ -16,19 +15,90 @@ const IndexPage: NextPage = () => (
         welcomeMessage="selamat datang"
```
Comment (resir014) [inline pages/index.tsx:15]:
```suggestion
        welcomeMessage="Selamat Datang"
```
Proper capitalisation.

File: pages/index.tsx [hunk 2]
```diff
@@ -16,19 +15,90 @@ const IndexPage: NextPage = () => (
...
-          </Link>
-        </p>
-      </PageBody>
+
+      <Box backgroundColor="#f2f2f2">
+        <Grid gridTemplateColumns="max-content 1fr" columnGap="5em" alignItems="center" py="6em" px="11%">
+          <Image display={['hidden', null, null, '']} size="400px" src="/rumah-komunitas/tshirt.jpg" alt="tshirt from rumah komunitas" />
+          <Grid>
+            <Heading as="h2" fontSize="xl" fontWeight={300} textTransform="uppercase">
+              kabar gembira!
```
Comment (resir014) [inline pages/index.tsx:24]:
```suggestion
              Kabar Gembira!
```

Proper capitalisation.

File: pages/index.tsx [hunk 3]
```diff
@@ -16,19 +15,90 @@ const IndexPage: NextPage = () => (
...
+              Dapatkan Segera
+            </LinkButton>
+          </Grid>
+        </Grid>
+      </Box>
+
+      <Box backgroundColor="lightBlue" paddingX="8.5em" paddingY="6em">
+        <Grid gap="2px" justifyContent="center" gridAutoFlow="row" textAlign="center">
+          <Heading as="h2" color="gray08" fontSize="20px" lineHeight="28px" fontWeight={300} textTransform="uppercase">
+            ingin belajar react?
```
Comment (resir014) [inline pages/index.tsx:43]:
```suggestion
            Ingin Belajar React?
```

Proper capitalisation.

File: pages/index.tsx [hunk 4]
```diff
@@ -16,19 +15,90 @@ const IndexPage: NextPage = () => (
...
+          </Heading>
+          <Heading as="h3" fontSize="48px" lineHeight="56px">
```
Comment (resir014) [inline pages/index.tsx:45]:
```suggestion
          <Heading as="h3" fontSize="md" fontWeight={600}>
```

Font weight should be 600. Proper type scale.

File: pages/index.tsx [hunk 5]
```diff
@@ -16,19 +15,90 @@ const IndexPage: NextPage = () => (
...
+            <LinkButton ml="-16px" href="https://www.rumahkomunitas.com/react-indonesia" isExternal>
+              Dapatkan Segera
+            </LinkButton>
+          </Grid>
+        </Grid>
+      </Box>
+
+      <Box backgroundColor="lightBlue" paddingX="8.5em" paddingY="6em">
+        <Grid gap="2px" justifyContent="center" gridAutoFlow="row" textAlign="center">
+          <Heading as="h2" color="gray08" fontSize="20px" lineHeight="28px" fontWeight={300} textTransform="uppercase">
```
Comment (resir014) [inline pages/index.tsx:42]:
```suggestion
          <Heading as="h2" color="gray08" fontSize="md" fontWeight={300} textTransform="uppercase">
```

Use consistent type scale.

File: pages/index.tsx [hunk 6]
```diff
@@ -16,19 +15,90 @@ const IndexPage: NextPage = () => (
...
-      </PageBody>
+
+      <Box backgroundColor="#f2f2f2">
+        <Grid gridTemplateColumns="max-content 1fr" columnGap="5em" alignItems="center" py="6em" px="11%">
+          <Image display={['hidden', null, null, '']} size="400px" src="/rumah-komunitas/tshirt.jpg" alt="tshirt from rumah komunitas" />
+          <Grid>
+            <Heading as="h2" fontSize="xl" fontWeight={300} textTransform="uppercase">
+              kabar gembira!
+            </Heading>
+            <Heading as="h3" fontSize="2.86rem" fontWeight={600}>
```
Comment (resir014) [inline pages/index.tsx:26]:
```suggestion
            <Heading as="h3" fontSize="2xl" fontWeight={600}>
```

Consistent font size.

File: pages/index.tsx [hunk 7]
```diff
@@ -16,19 +15,90 @@ const IndexPage: NextPage = () => (
...
-            <a>About</a>
-          </Link>
-        </p>
-      </PageBody>
+
+      <Box backgroundColor="#f2f2f2">
+        <Grid gridTemplateColumns="max-content 1fr" columnGap="5em" alignItems="center" py="6em" px="11%">
+          <Image display={['hidden', null, null, '']} size="400px" src="/rumah-komunitas/tshirt.jpg" alt="tshirt from rumah komunitas" />
+          <Grid>
+            <Heading as="h2" fontSize="xl" fontWeight={300} textTransform="uppercase">
```
Comment (resir014) [inline pages/index.tsx:23]:
```suggestion
            <Heading as="h2" fontSize="md" fontWeight={300} textTransform="uppercase">
```

Proper type scale.

File: pages/index.tsx [hunk 8]
```diff
@@ -16,19 +15,90 @@ const IndexPage: NextPage = () => (
...
+                      </Text>
+                      <LinkButton
+                        _hover={undefined}
+                        _groupHover={{ backgroundColor: 'tangaroa' }}
+                        as="span"
+                        mt="auto"
+                        backgroundColor="astronautBlue"
+                        color="white"
+                      >
+                        Kunjungin Situs
```
Comment (resir014) [inline pages/index.tsx:88]:
```suggestion
                        Kunjungi Situs
```

`Kunjungin` -> `Kunjungi`

**Review by resir014 — CHANGES_REQUESTED** [review]
Ada changes, mostly buat nyesuaiin font size sama line height aja:

**Comment by adibfirman** [comment]
Thanks @resir014 , ku udah coba update dengan type-scale

**Review by resir014 — APPROVED** [review]

**Pull request merged**